index.html

```diff
@@ -585,6 +584,55 @@
       </div>
     </div>
 
+    <!-- Quality assurance of software-defined cloud infrastructure - Ahmed Zerouali -->
+    <div class="portfolio-modal modal fade" id="ahmedzerouali" tabindex="-1" role="dialog" aria-hidden="true">
+      <div class="modal-content">
+        <div class="close-modal" data-dismiss="modal">
+          <div class="lr">
+            <div class="rl">
+            </div>
+          </div>
+        </div>
+        <div class="container">
+          <div class="row">
+            <div class="modal-body">
+              <h3>Ahmed Zerouali</h3>
+              <h4>Quality assurance of software-defined cloud infrastructuret</h4>
+              ahmed.zerouali@umons.ac.be, ahmed@bitergia.com 
+              <hr>
+              <div class="row">
+                <div class="col-sm-4 portfolio-item">
+                  <h4>Bio</h4>
+                  Ahmed Zerouali is a Researcher and PhD candidate at the University of Mons and working within the European SENECA project, at Universidad Rey Juan Carlos and Bitergia, Madrid.
```

Comment:
Please add Bitergia as the place you're working at. Universidad Rey Juan Carlos/Mons is where you'll get your PhD :).